### ISSUE TITLE
Scheduled dark mode update doesn't trigger wallpaper update 

### ISSUE THREAD

**Issue by nicovong** [Yanndroid/DualWallpaper#15]
Hello thanks for this beautiful app.
It works when I manually switch the dark mode, but it doesn't with the scheduled dark mode. I'm using automatic dark mode on Samsung One UI 5.1.1.
![Screenshot_20231210_185615_Settings](https://github.com/Yanndroid/DualWallpaper/assets/30236832/87693e1e-de41-47d4-b1ed-dd41b2124b56)

**Comment by nicovong** [comment]
It seems to work properly now, I don't understand why, so I close the issue.

**Comment by YuenSzeHong** [comment]
same for miui 14, seem it is a universal issue

**Comment by Yanndroid** [comment]
I think it's an issue on newer Android versions. The battery optimization is pretty aggressive these days. You can try to set it to "Unrestricted" in settings. On Samsung it's in App Info > Battery. I might not change anything though. The time trigger might be more reliable.

**Comment by nicovong** [comment]
Yes I already tried to disable battery optimization, but it has no impact

![Screenshot_20240211_193856_Settings.jpg](https://github.com/Yanndroid/DualWallpaper/assets/30236832/abc2ef5c-0270-4b66-a0f2-ce3a8401eafa)

**Comment by YuenSzeHong** [comment]
@![Screenshot_2024-02-12-17-55-22-104_com.miui.powerkeeper.jpg](https://github.com/Yanndroid/DualWallpaper/assets/40477634/0e2bb92e-bb8e-440b-9c7c-52e91e1af37e)

same here (sorry for Chinese UI)

**Comment by LiviYan** [comment]
I had a similar problem and what fixed it for me was to go into App management > Dual Wallpaper > Battery usage > Allow background activity. This was on a OnePlus ace 3, which is basically just Android 14. Hopefully that helps someone.
